packages/ui/src/components/editor/properties/trigger/index.tsx

```diff
@@ -85,7 +84,7 @@ const TriggerProperties: EditorComponentType = (props) => {
       })
     }
     targets.set(options)
-  }, [callbackQuery])
+  }, [JSON.stringify(callbackQuery)])
```

Comment:
This stringing shouldn't be necessary, since there is not something that would change the query inside the useEffect. Either useQuery is broken or something else is causing a feedback here.